README.md

```diff
@@ -1,2 +1,23 @@
+Status: Currently in draft and non-binding. 
+
 # Argo Conformance Program
-A standard to ensure interoperability, protect users, and prevent surprises when using commercial offerings with Argo in the name. Currently in draft and non-binding. 
++The Argo community continues to grow dramatically – adding new users, new contributors, and new capabilities. The Argo Conformance Program creates incentives and requirements designed to help the ecosystem grow in a sustainable and user-friendly way. As a diverse set of tools grows around Argo, users can trust that products certified as compliant will interoperate with each other and be up-to-date with the latest open source releases. This ensures a consistent user experience between products and flexibility to choose between vendors.
+
+When users use an Argo certified offering they should be able to trust that offering to work with the open source versions of Argo and that configuration designed for Argo will always work.
+
+Software conformance ensures predictability and helps protect the Argo brand. 
+
+# Goals
++* Guarantee a consistent and predictable user experience
++* Guarantee interoperability, including API compatibility
++* Guarantee conformant products are up-to-date with latest Argo releases 
++* Ensure contributions from commercial partners
+ * Grow and strengthen the Argo community
++* Protect the Argo brand
+
+
+# Benefits of Certification
+In addition to giving users confidence and protecting the Argo community certified offerings may
+* Advertise compliance and certification
+* Use Argo logo and branding in marketing materials
+* Exposure to community
```

Comment:
```suggestion
* Exposure to community

# Benefits to Community
* Promote interoperability and compatibility between Argo products
* Discourage forking of the Argo code base
* Encourage commercial interests to work together to grow the Argo ecosystem rather than purely on competition with each other
```

Reply:
I like it!